lib/device-log/ios-device-log.js

```diff
@@ -0,0 +1,45 @@
+import { logger } from 'appium-support';
+import { IOSLog } from './ios-log';
+import { services } from 'appium-ios-device';
+
+const log = logger.getLogger('IOSDeviceLog');
+
+class IOSDeviceLog extends IOSLog {
+
+  constructor (opts) {
+    super();
+    this.udid = opts.udid;
+    this.showLogs = !!opts.showLogs;
+    this.service = null;
+  }
+
+  async startCapture () {
+    if (this.service) {
+      return;
+    }
+    this.service = await services.startSyslogService(this.udid);
+    this.service.start(this.onLog.bind(this));
+  }
+
+  onLog (logLine) {
+    this.broadcast(logLine);
+    if (this.showLogs) {
+      log.info(logLine);
```

Comment:
Is there nothing to distinguish log level?

Reply:
We can parse it but it is messy

Reply:
Meh. No need.